tokens/finn.no/card.yml

```diff
@@ -30,9 +30,8 @@ color:
         hover: s-color-border-hover
         active: s-color-border-active
         selected:
-          _: s-color-background-primary
-          hover: s-color-background-primary-hover
-          active: s-color-background-primary-active
```

Comment:
Maybe we could move unused tokens to `deprecated.yml` in case someone does rely on this token, e.g. they're using some old version of finn-no.css, who knows.

Reply:
I'll reintroduce them here for backwards compatibility, with a comment to be removed, makes it easier to merge with the v2 changes.